comic_dl/sites/readcomicOnlineto.py

```diff
@@ -98,16 +98,25 @@ def full_series(self, comic_url, comic_name, sorting, download_directory, chapte
         # print(listing_table)
 
         for elements in listing_table:
-            x = elements.findAll('a')
-            for a in x:
-                all_links.append(str(a['href']).strip())
+            x = elements.findAll('tr')
+            unorderer = []
+            for row in x:
+                colums = row.findAll('td')
+                if len(colums) == 0:
+                    continue
+                unorderer.append({
+                    "date": str(colums[1].contents[0]).strip(),
+                    "link": str(colums[0].find('a')['href']).strip()
+                })
+            # order links by date, in reverse order because of readcomiconline logic
+            unorderer.sort(key=lambda x: datetime.strptime(x['date'], '%m/%d/%Y'))
+
+            all_links = map(lambda x: x['link'], unorderer)
 
         """Readcomiconline.to shows the chapters in the Descending order. The 1st chapter is at the bottom, hence, at
            the end of the list. So, we'll reverse the list, to perform the ranging functionality properly.
            This is a fix for issues like #74.
         """
```

Comment:
remove the previous comment, unneeded now

Reply:
good call!